Database-Files/CreateDatabase.sql

```diff
@@ -126,3 +113,20 @@ CREATE TABLE resp_Role (
     FOREIGN KEY (respID) REFERENCES responsibilities(ID),
     FOREIGN KEY (roleID) REFERENCES jobRole(ID)
 );
+
+
+DROP TABLE IF EXISTS authData;
+CREATE TABLE authData (
+    localID INT PRIMARY KEY AUTO_INCREMENT,
+    username VARCHAR(50) NOT NULL UNIQUE,
+    passwordHash VARCHAR(60) NOT NULL,
+    failedAttempts INT NOT NULL DEFAULT 0,
+    lockedOut BOOL NOT NULL DEFAULT FALSE,
+    lockoutDate VARCHAR(30) DEFAULT NULL,
+    jwt VARCHAR(200),
+    jwtDate VARCHAR(30) DEFAULT NULL,
+    isAdmin BOOLEAN NOT NULL DEFAULT 0,
+    ID INT,
```

Comment:
what is this ID for? its not really clear from the name

Reply:
This is a foreign key for the id field in the userData table, we have done it this way as since the userData and authData are in separate tables, we want to keep the information data and security data synchronised

Reply:
need to change ID to be a better representative name